hw/ip/otbn/doc/_index.md

```diff
@@ -426,11 +426,73 @@ Rough expected process:
 
 ## Error conditions
 
-<div class="bd-callout bd-callout-warning">
-  <h5>Note</h5>
-
-  To be filled in as we create the implementation.
-</div>
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Code</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>No Error</td>
+      <td>0x00</td>
+      <td>Execution was successful</td>
+    </tr>
+    <tr>
+      <td>Bad Data Address</td>
+      <td>0x01</td>
+      <td>
+      A Dmem read or write occurred with an out of bounds or unaligned address
+      </td>
+    </tr>
+    <tr>
+      <td>Bad Instruction Address</td>
+      <td>0x02</td>
+      <td>
+      An instruction fetch occurred with an out of bounds or unaligned address
+      </td>
+    </tr>
+    <tr>
+      <td>Call Stack over/underflow</td>
```

Comment:
Nit: I think under/over is probably more common than over/under.

Reply:
Indeed it is, had done it that way as my initial description referred to them in the same order, but I ended up writing things differently. I'll switch back to under/over.